unix/snowagent.config

```diff
@@ -28,7 +28,15 @@
     <Software>
         <!-- Only if Include is specified a disk scan will be initiated. -->
         <Include>
-            <Path recursive="true" unconditionally="false">/*</Path>
+            <!-- Only include necessary folders for the file scan-->
+            <Path recursive="true" unconditionally="false">/etc/*</Path>
+			<Path recursive="true" unconditionally="false">/var/*</Path>
```

Comment:
can we keep the same indentation?

Reply:
On notepad++ it is using the same identation. Don't know why GitHub has it weird like that

Reply:
tabs vs spaces.

Reply:
same as above